projects/schematics/tsconfig.schematics.spec.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "./tsconfig.schematics.json",
```

Comment:
This config doesn't add anything new that the schematics tsconfig doesn't have. I think we can drop it.